### ISSUE TITLE
Cannot create new server - mysqli_connect() expects parameter 4 to be string, null given

### ISSUE THREAD

**Issue by kckanav** [phpmyadmin/phpmyadmin#16536]
```
Fatal error: Uncaught TypeError: mysqli_connect() expects parameter 4 to be string, null given in /nfs/bronfs/uwfs/hw00/d22/kckanav/phpMyAdmin-5.0.4-all-languages/libraries/classes/Config/Validator.php:226 Stack trace: #0 /nfs/bronfs/uwfs/hw00/d22/kckanav/phpMyAdmin-5.0.4-all-languages/libraries/classes/Config/Validator.php(226): mysqli_connect('kckanav.ovid.u....', 'root', 'allNerds12x', NULL, 1707, NULL) #1 /nfs/bronfs/uwfs/hw00/d22/kckanav/phpMyAdmin-5.0.4-all-languages/libraries/classes/Config/Validator.php(304): PhpMyAdmin\Config\Validator::testDBConnection('kckanav.ovid.u....', 1707, NULL, 'root', 'allNerds12x', 'Server') #2 [internal function]: PhpMyAdmin\Config\Validator::validateServer('Server', Array) #3 /nfs/bronfs/uwfs/hw00/d22/kckanav/phpMyAdmin-5.0.4-all-languages/libraries/classes/Config/Validator.php(142): call_user_func_array('PhpMyAdmin\\Conf...', Array) #4 /nfs/bronfs/uwfs/hw00/d22/kckanav/phpMyAdmin-5.0.4-all-languages/libraries/classes/Config/FormDisplay.php(201): PhpMyAdmin\Config\Validator::validate in /nfs/bronfs/uwfs/hw00/d22/kckanav/phpMyAdmin-5.0.4-all-languages/libraries/classes/Config/Validator.php on line 226
```

THIS IS THE ERROR I AM GETTING
It seems like the @mysqli_connect does not take null parameter, but on line 226 in Validator.php the 4th parameter is hard coded to be NULL.

**Comment by williamdes** [comment]
Fixed by f2e49a92c6681414b74140d98ea92607db791c22 (https://github.com/phpmyadmin/phpmyadmin/commit/f2e49a92c6681414b74140d98ea92607db791c22#diff-08d19463f2535e34c30108f532ad5190b89b5aefab8ce407c8905d2b4820e206R232)

Already answered on: #16537

**Comment by williamdes** [comment]
Duplicate of #16537
